frontend/src/components/Search/CourseConflictIcon.tsx

```diff
@@ -47,26 +56,28 @@ function CourseConflictIcon({ course }: { readonly course: Listing }) {
     [course, data],
   );
 
+  const seasonMismatch = course.season_code !== CUR_SEASON;
+
+  const displayConflict = inModal
+    ? seasonMismatch
+    : conflicts.length > 0 || seasonMismatch;
+
   return (
-    // Smooth fade in and out transition
-    <Fade in={conflicts.length > 0}>
+    <Fade in={displayConflict}>
       <div>
-        {conflicts.length > 0 && (
+        {displayConflict && (
           <OverlayTrigger
             placement="top"
             overlay={(props) => (
               <Tooltip {...props} id="conflict-icon-button-tooltip">
                 <small style={{ fontWeight: 500 }}>
-                  Conflicts with: <br />
-                  {conflicts.map((x) => x.course_code).join(', ')} <br />
+                  {seasonMismatch
+                    ? `This will add this course to a worksheet in a different season.`
```

Comment:
Not "in a different season"; we always add to the season the course is in. It's just not the "current season".